ui/components/app/collectibles-detection-notice/index.scss

```diff
@@ -1,31 +1,52 @@
 .collectibles-detection-notice {
+  margin: 16px 16px 0 16px;
+
   &__message {
     position: relative;
-    padding: 0 1rem 1rem 1rem !important;
-
-    &__close-button {
-      background-color: transparent;
-
-      &::after {
-        position: absolute;
-        content: '\00D7';
-        font-size: 29px;
-        font-weight: 200;
-        color: var(--black);
-        background-color: transparent;
-        top: 0;
-        right: 12px;
-        cursor: pointer;
+    padding: 0.75rem 0.75rem 1rem 0.75rem !important;
+
+    & &__close-button {
+      color: var(--color-icon-default);
+      background: none;
+      position: absolute;
+      cursor: pointer;
+      right: 8px;
+
+      @media screen and (min-width: 1439px) {
+        display: none;
+      }
+    }
+
+    & &__data {
+      @media screen and (min-width: 1439px) {
+        display: inline-flex;
       }
     }
 
-    a.collectibles-detection-notice__message__link {
-      @include H6;
+    &__fullscreen-links {
+      @media screen and (min-width: 1439px) {
+        margin-left: calc((100vw - 95vw) / 2);
```

Comment:
Bit weird values for left-margin.

Reply:
@jpuri I have updated the margin, but I was not able to test it out on a bigger screen as this `min-width` is almost the max size of my screen.